modulemd/meson.build

```diff
@@ -1,42 +1,41 @@
 modulemd_srcs = [
-    'modulemd.c',
-    'modulemd.h',
-    'modulemd-modulemetadata.c',
-    'modulemd-modulemetadata.h',
+  'modulemd-modulemetadata.c',
+]
+modulemd_hdrs = [
+  'modulemd.h',
+  'modulemd-modulemetadata.h',
 ]
 
 modulemd_lib = library(
-    'modulemd',
-    sources : modulemd_srcs,
-    dependencies : [
-        gobject,
-    ],
-    install : true,
-    soversion: '0',
-    version: '0.1.0',
-
+  'modulemd',
+  sources : modulemd_srcs,
+  dependencies : [
+      gobject,
+  ],
+  install : true,
+  soversion: '0',
+  version: '0.0.0',
```

Comment:
I planned to make separate commit, but looks like I sneaked it to big one =(

well, it's just cosmetic thing. Feel free to ignore it.